http/src/main/resources/init-resources/msdsvmcontent/command_no_public.txt

```diff
@@ -0,0 +1,12 @@
+az group deployment create --subscription <SUBSCRIPTION ID> ^
```

Comment:
I don't think the arm templates and this file are needed here becuz they won't be used...But we can keep them in https://github.com/DataBiosphere/terra-azure-arm-templates just for record